packages/@lwc/integration-karma/test/template/directive-external/custom-elements/ce-with-children.js

```diff
@@ -0,0 +1,16 @@
+if (!process.env.COMPAT) {
+    class CEWithChildren extends HTMLElement {
+        constructor() {
+            super();
+            this.attachShadow({ mode: 'open' });
+            this.shadowRoot.innerHTML = `
+            <h1>Test h1</h1>
+            <div>
+                <p>Test p</p>
+            </div>
+            <slot></slot>
+            `;
+        }
+    }
+    'customElements' in window && customElements.define('ce-with-children', CEWithChildren);
```

Comment:
```suggestion
    customElements.define('ce-with-children', CEWithChildren)
```

I'm assuming the `customElements` check is not necessary since we're not in COMPAT?